### ISSUE TITLE
nuxt problem cache in vuex

### ISSUE THREAD

**Issue by davizxtr** [nuxt/nuxt.js#7088]
I use nuxt vuex for user login.

I have nuxtServerInit to get cache from the server. For example, I use it to load basic categories for the web.

The problem is:
1. A user logs in, all right.
2. Log out (The token is removed and refreshtoken and the store is cleaned)
3. The page is refreshed with F5 refresh and the user appears again!
4. Ctrl + F5 and it still appears! The store information is not deleted.

I don't have any cache plugin installed, I don't use workbox, I don't use PWA.

`actions: {
      async nuxtServerInit ({ state,dispatch,getters }, { req, $axios }) {
        console.log('nuxtServerInit :: ', state.user)
        await dispatch('FETCH_INIT_CACHE_ACTION')
      }
    },`

![nuxt issue](https://user-images.githubusercontent.com/37248385/76798842-a8bc6980-67d0-11ea-860d-134c12e11c9c.jpg)

**Comment by Atinux** [comment]
Can you please create a reproduction on CodeSandBox (https://template.nuxtjs.org) and open a bug report?

**Comment by davizxtr** [comment]
@Atinux I've solved it. 

I don't know what the problem was really... 
I've updated the classic store to module style.
I had a problem with the console and the current configuration of my store was in disuse and nuxt 3 was removed... so I decided to update it.

It works perfectly and it has not given me any problem again.

Thanks anyway for answering.
By the way, when is nuxt version 3 expected to be released?
